integration-test/src/python/local_test_runner/main.py

```diff
@@ -190,34 +195,35 @@ def submitTopology(heronCliPath, testCluster, testJarPath, topologyClassPath,
       '%s' % (outputFile),
       '%d' % (len(TEST_INPUT))
   ]
-  logging.info("Submitting topology: ")
+  logging.info("Submitting topology: %s", splitcmd)
   logging.info(splitcmd)
   p = subprocess.Popen(splitcmd)
   p.wait()
-  logging.info("Submitted topology")
+  logging.info("Submitted topology %s", topologyName)
 
 def killTopology(heronCliPath, testCluster, topologyName):
   ''' Kill a topology using heron-cli '''
-  logging.info("Killing topology")
   splitcmd = [
       '%s' % (heronCliPath),
       'kill',
       '--verbose',
       '%s' % (testCluster),
       '%s' % (topologyName),
   ]
-  logging.info("Killing topology:")
+  logging.info("Killing topology: %s", splitcmd)
   logging.info(splitcmd)
```

Comment:
`logging.info("Killing topology: %s", splitcmd)`

and again down below

`logging.info(splitcmd)`